classes/classes/Creature.as

```diff
@@ -2462,7 +2462,17 @@ package classes
 		{
 			return skinType == SKIN_TYPE_PLAIN;
 		}
-
+		
+		public function hasFleece():Boolean
+		{
+			return skinType == SKIN_TYPE_FLEECE;
+		}
+		
+		public function hasFurOrFleece():Boolean
+		{
+			return hasFur() || hasFleece();
+		}
```

Comment:
I'd suggest to refactor that function to be named isFurry() so we don't need to rename it every time, when we add more skintypes. `hasFurOrFleeceOrBristlesOrWhatever` would sound kinda awkward anyway 😕